backend/utils.py

```diff
@@ -33,3 +43,18 @@ def create_transaction(db):
             pass
 
     return db.transaction()
+
+
+class AppLogger(object):
+
+    @staticmethod
+    def log_message(log_fn, args, response):
+        log_fn(
+            f"{datetime.utcnow().strftime('%d %b, %Y, %H:%M:%S')} - "
```

Comment:
can you double check if you need. to log the time, the logger does that automatically. If you want to control the complete log formatting, you can use `logging.Formatter`

Reply:
This is what shows in the log now (no time without adding it in the code) - 
`INFO:root:04 May, 2022, 19:33:55 - POST - /banner - Local Dev - local-dev@testing.com - {"message": "Testing banner", "severity": "info"} - Banner created`